### PR TITLE
Add WebP image format

### PR DESCRIPTION
**Pull request by xchs**
Add WebP image format to the list of allowed upload and download file types.

### CODE REVIEW

**Comment by leofeyer** [comment]
If @ausi confirms this change, the PR needs to be rebased onto the `4.8` branch.

**Comment by xchs** [comment]
> the PR needs to be rebased onto the `4.8` branch

So do you consider this a bugfix, right?

**Comment by leofeyer** [comment]
If @ausi has not deliberately left out `webp` at this place, then yes.

File: core-bundle/src/Resources/contao/config/default.php [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 // File uploads
 $GLOBALS['TL_CONFIG']['uploadTypes']
-	= 'jpg,jpeg,gif,png,ico,svg,svgz,'
+	= 'jpg,jpeg,gif,png,ico,svg,svgz,webp'
```
Comment (ausi) [inline core-bundle/src/Resources/contao/config/default.php]: Missing comma

```suggestion
	= 'jpg,jpeg,gif,png,ico,svg,svgz,webp,'
```

File: core-bundle/src/Resources/contao/config/default.php [hunk 2]
```diff
@@ -89,7 +89,7 @@
 $GLOBALS['TL_CONFIG']['defaultChmod']         = array('u1', 'u2', 'u3', 'u4', 'u5', 'u6', 'g4', 'g5', 'g6');
 $GLOBALS['TL_CONFIG']['editableFiles']        = 'css,csv,html,ini,js,json,less,md,scss,svg,svgz,txt,xliff,xml,yml,yaml';
 $GLOBALS['TL_CONFIG']['allowedDownload']
-	= 'jpg,jpeg,gif,png,svg,svgz,'
+	= 'jpg,jpeg,gif,png,svg,svgz,webp'
```
Comment (ausi) [inline core-bundle/src/Resources/contao/config/default.php]:
```suggestion
	= 'jpg,jpeg,gif,png,svg,svgz,webp,'
```

**Review by ausi — CHANGES_REQUESTED** [review]
Aside from the missing commas this looks good to me 👍

**Comment by ausi** [comment]
Followup pull request: #829

**Pull request closed without merging**